game/script-topics.rpy

```diff
@@ -1537,6 +1545,68 @@ label talk_driving:
             else:
                 n "That's what friends are for, [player]!"
 
+    return
+
+# Natsuki laments her inability to drive and questions the player on if they can
+init 5 python:
+    registerTopic(
+        Topic(
+            persistent._topic_database,
+            label="talk_sustainable_fashion",
+            unlocked=True,
+            prompt="Sustainable fashion",
+            conditional=None,
+            category=["Environment", "Fashion"],
+            nat_says=True,
+            affinity_range=(jn_affinity.UPSET, jn_affinity.LOVE),
+            location="classroom"
+        ),
+        topic_group=TOPIC_TYPE_NORMAL
+    )
+
+label talk_sustainable_fashion:
+    n "Hey,{w=0.1} [player]..."
+    n "This is kinda random,{w=0.1} but are you into fashion?"
+    if jn_affinity.get_affinity_state() >= store.jn_affinity.HAPPY:
+        n "I know I am!{w=0.2} Can you tell?"
+        n "Ehehe."
+
+    else:
+        n "I know I am."
+
+    n "But what caught me by surprise is just how much waste there is."
+    n "Seriously,{w=0.1} [player] {w=0.1}-{w=0.1} it's insane!"
+    n "People throw away a {i}lot{/i} of clothing...{w=0.3} it's estimated that we discard around 90{w=0.3} {i}million{/i}{w=0.3} tonnes every year."
```

Comment:
Maybe use a little something other than `discard` here, I think I can see Monika or maybe more Yuri using that in this context, but maybe `get rid of` might fit Natsuki a little more?